packages/neovim/neovim.pacscript

```diff
@@ -0,0 +1,24 @@
+name="neovim"
+pkgname="neovim"
+version="5.0.0"
+url="https://github.com/neovim/neovim/archive/refs/tags/v0.5.0.tar.gz"
```

Comment:
`version=0.5.0`, not 5.0.0
I think `url` should be `https://github.com/neovim/neovim/archive/refs/tags/v${version}.tar.gz` to facilitate updating the package.

Reply:
> I think url should be https://github.com/neovim/neovim/archive/refs/tags/v${version}.tar.gz to facilitate updating the package.

It's not mandatory to use `${version}`, but it makes it easier for the maintainer to update the package, since they have to update only the `version` variable then, and the `url` gets automatically updated.